test/index.js

```diff
@@ -0,0 +1,57 @@
+const proxyquire = require('proxyquire').noCallThru()
+const assert = require('assert')
+
+
+describe('main', () => {
+  describe('getStreams returns data from AWS', () => {
+    let main
+
+    before(() => {
```

Comment:
@crccheck TBTH the way you've done it is fine, Proxyquire + aws-sdk is a much better option than mocking aws-sdk using, say, Sinon. If not wanting to use Proxyquire, the lib I mentioned is a decent option.